index.html

```diff
@@ -153,30 +146,32 @@ <h2>
       </h2>
       <p>
         The {{Document}} of the <a>top-level browsing context</a> can be in one
-        of the following <dfn data-export="" data-lt=
-        "visibility state">visibility states</dfn>:
+        of the following <dfn data-dfn-for="Document" data-export="">visibility
```

Comment:
Maybe the exported term should be singular. 
```suggestion
        of the following <dfn data-dfn-for="Document" data-export="" data-lt="visibility state">visibility
```